pkg/util/ip_reservation_test.go

```diff
@@ -89,6 +90,7 @@ func TestParseCIDR(t *testing.T) {
 func TestGetUnReservedIPRange(t *testing.T) {
 	// Using IPs which are not the beginning IPs of /29 CIDRs to evaluate the edge case
 	ips := [8]string{"192.168.92.3/29", "192.168.92.10/29", "192.168.92.20/29", "192.168.92.28/29"}
+	ipRangeSize := 29
```

Comment:
use the constant that was defined in ip_defs?

Lets add test case for 24 also.